### PR TITLE
Display Schema object names in the table of contents menu

### PR DESCRIPTION
**Pull request by wesrice**

### CODE REVIEW

File: content/minfraud/api-documentation/examples.mdx [hunk 1]
```diff
@@ -0,0 +1,217 @@
+export const request = `
```
Comment (kevcenteno) [inline content/minfraud/api-documentation/examples.mdx:1]: I don't think this file is used anywhere

Comment (wesrice) [reply content/minfraud/api-documentation/examples.mdx]: Ah, this was an artifact from an idea I was exploring. I removed this file.

File: src/gatsby/gatsby-node/createResolvers.ts [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+import getTocItems from '../../utils/get-toc-items';
+
+export const createResolvers: GatsbyNode['createResolvers'] = async(
+  args: CreateResolversArgs,
+): Promise<any> => {
+  const { createResolvers } = args;
+  const {
+    store,
+    pathPrefix,
```
Comment (kevcenteno) [inline src/gatsby/gatsby-node/createResolvers.ts:17]: You can get rid of all of these variables except for `store` and `...helpers`

File: src/gatsby/gatsby-node/createResolvers.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+    getNode,
+    getNodes,
+    cache,
+    reporter,
+    actions,
+    schema,
+    ...helpers
+  } = args;
+
+  const plugins = args.store.getState().flattenedPlugins;
```
Comment (kevcenteno) [inline src/gatsby/gatsby-node/createResolvers.ts:27]: `s/args.store/store/`

File: src/gatsby/gatsby-node/createResolvers.ts [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+
+  const mdxPlugin = plugins.find(
+    (plugin: any) => plugin.name === 'gatsby-plugin-mdx'
+  );
+
+  const processMDX = ({ node }: any) =>
+    genMDX({
+      actions,
```
Comment (kevcenteno) [inline src/gatsby/gatsby-node/createResolvers.ts:35]: If the above cleanup is implemented, you could reduce this to `{node, options: mdxPlugin.pluginOptions, store, ...helpers}`

**Review by kevcenteno — CHANGES_REQUESTED** [review]
There is also a build and test failure on this branch

**Comment by wesrice** [comment]
@kevcenteno I made updates based on your recommendations. I also added some comments to what this code actually does, just because it is a very roundabout/atypical way of producing something that, on the surface, should be a lot simpler.

**Pull request merged**